lib/__tests__/dom.nested.test.js

```diff
@@ -51,7 +51,9 @@ describe('DocumentMeta - DOM nested', () => {
       <div>
         <DocumentMeta {...DOC_META} />
         <div>
-          <DocumentMeta {...DOC_META_NESTED} extend />
+          <DocumentMeta {...DOC_META_NESTED} extend>
+            <div>Hello world!</div>
```

Comment:
It’s not visual but it’s there and it’s bugging me, because I know it’s there.
The other way I thought of fixing this was not passing children into the clone function but that seems somewhat hacky to me.

Thanks!